src/main/resources/application.properties

```diff
@@ -8,5 +8,5 @@ spring.jpa.show-sql=true
 spring.jpa.open-in-view=false
 
 jwt.expiration.minutes=30
-jwt.expiration.hours=1
+jwt.expiration.hours=10
```

Comment:
Don’t  forget to change it after testing )

Reply:
yes, but i think on a real website 10 hours it is normal, no one wants to relogin every hour

Reply:
Not fixed. 
FYI You relogin on PrivateBank site almost every hour)